packages/@react-spectrum/label/src/Field.tsx

```diff
@@ -25,8 +26,15 @@ function Field(props: SpectrumFieldProps, ref: RefObject<HTMLElement>) {
     isRequired,
     necessityIndicator,
     includeNecessityIndicatorInAccessibilityName,
+    validationState,
+    description,
```

Comment:
Is this correct that `Field` should accept `description, errorMessage` and also `descriptionProps, errorMessageProps`, then pass them all down to `HelpText`?

Reply:
This seems correct as well, is there something you are concerned about here?

Reply:
Thanks, no concern, just wanted to make sure that made sense.